File: src/Agents.Service/Queries/Distributions/CommissionQuery.cs
using System;
using System.ComponentModel.DataAnnotations;
using Util;
using Util.Datas.Queries;

namespace Agents.Service.Queries.Distributions {
    /// <summary>
    /// 佣金查询参数
    /// </summary>
    public class CommissionQuery : QueryParameter {
        /// <summary>
        /// 佣金标识
        /// </summary>
        [Display(Name="佣金标识")]
        public Guid? CommissionId { get; set; }
        /// <summary>
        /// 代理标识
        /// </summary>
        [Display(Name="代理标识")]
        public Guid? AgentId { get; set; }
        /// <summary>
        /// 订单标识
        /// </summary>
        [Display(Name="订单标识")]
        public Guid? OrderId { get; set; }
        /// <summary>
        /// 类型
        /// </summary>
        [Display(Name="类型")]
        public int? Type { get; set; }
        /// <summary>
        /// 金额
        /// </summary>
        [Display(Name="金额")]
        public decimal? Money { get; set; }
        /// <summary>
        /// 状态
        /// </summary>
        [Display(Name="状态")]
        public int? State { get; set; }
        
        private string _note = string.Empty;
        /// <summary>
        /// 备注
        /// </summary>
        [Display(Name="备注")]
        public string Note {
            get => _note == null ? string.Empty : _note.Trim();
            set => _note = value;
        }
        /// <summary>
        /// 起始创建时间
        /// </summary>
        [Display( Name = "起始创建时间" )]
        public DateTime? BeginCreationTime { get; set; }
        /// <summary>
        /// 结束创建时间
        /// </summary>
        [Display( Name = "结束创建时间" )]
        public DateTime? EndCreationTime { get; set; }
        /// <summary>
        /// 创建人
        /// </summary>
        [Display(Name="创建人")]
        public Guid? CreatorId { get; set; }
        /// <summary>
        /// 起始最后修改时间
        /// </summary>
        [Display( Name = "起始最后修改时间" )]
        public DateTime? BeginLastModificationTime { get; set; }
        /// <summary>
        /// 结束最后修改时间
        /// </summary>
        [Display( Name = "结束最后修改时间" )]
        public DateTime? EndLastModificationTime { get; set; }
        /// <summary>
        /// 最后修改人
        /// </summary>
        [Display(Name="最后修改人")]
        public Guid? LastModifierId { get; set; }
    }
}

File: src/Agents.Admin/Typings/app/distributions/commission/model/commission-query.ts
import { QueryParameter } from '../../../../util';

/**
 * 佣金查询参数
 */
export class CommissionQuery extends QueryParameter {
    /**
     * 佣金标识
     */
    commissionId;
    /**
     * 代理标识
     */
    agentId;
    /**
     * 订单标识
     */
    orderId;
    /**
     * 类型
     */
    type;
    /**
     * 金额
     */
    money;
    /**
     * 状态
     */
    state;
    /**
     * 备注
     */
    note;
    /**
     * 起始创建时间
     */
    beginCreationTime;
    /**
     * 结束创建时间
     */
    endCreationTime;
    /**
     * 创建人
     */
    creatorId;
    /**
     * 起始最后修改时间
     */
    beginLastModificationTime;
    /**
     * 结束最后修改时间
     */
    endLastModificationTime;
    /**
     * 最后修改人
     */
    lastModifierId;
}